docs/Contributing/API-for-contributors.md

```diff
@@ -1357,6 +1358,13 @@ If the `name` is not already associated with an existing team, this API route cr
         }
       },
       "scripts": ["path/to/script.sh"],
+      "software": [
+        {
+          "url": "https://cdn.zoom.us/prod/5.16.10.26186/x64/ZoomInstallerFull.msi",
+          "pre_install_query": "SELECT 1 FROM macos_profiles WHERE uuid='c9f4f0d5-8426-4eb8-b61b-27c543c9d3db';",
+          "post_install_script": "sudo /Applications/Falcon.app/Contents/Resources/falconctl license 0123456789ABCDEFGHIJKLMNOPQRSTUV-WX",
+        }
+      ]
```

Comment:
note that as with scripts, profiles, etc we need yet another endpoint to "batch apply" them.

https://github.com/fleetdm/fleet/blob/e5283ef2a819aff83d86a7c7ffceaea1495970ed/docs/Contributing/API-for-contributors.md?plain=1#L589-L591

https://github.com/fleetdm/fleet/blob/e5283ef2a819aff83d86a7c7ffceaea1495970ed/docs/Contributing/API-for-contributors.md?plain=1#L2738-L2744

I will go with something similar.

Reply:
Hey @roperzh, I've updated PR to include software batch-apply [here](https://github.com/fleetdm/fleet/pull/17865/files#diff-831a63ebb3cab9b4e6b82d803d9ffcdc3722b12d486dcd7b13e9576643bb50b0R2844). Let me know what do you think?